### ISSUE TITLE
`emsdk install 2.0.20-lto` does not work

### ISSUE THREAD

**Issue by donalffons** [emscripten-core/emscripten#14156]
Hi,

I already posted this as a reply to [your Tweet](https://twitter.com/emscripten/status/1392149058945314819) - so please forgive me for posting this twice, but I'm really excited about this news. I would like to test the new 2.0.20-lto builds and see if I can reduce my build times, which are currently at more than 24 hours for one of my build setups.

Unfortunately I get the following error when trying the command from the tweet:

```bash
$ emsdk install 2.0.20-lto
Fetching emscripten-releases repository...
Repository 'https://chromium.googlesource.com/emscripten-releases' already cloned to directory '/usr/lib/emsdk/releases', skipping.
Fetching latest changes to the branch 'master' for '/usr/lib/emsdk/releases'...
Already up to date.
Successfully updated and checked out branch 'master' on repository '/usr/lib/emsdk/releases'
Current repository version: "Tue, 11 May 2021 19:36:36 +0000 f4564474f39d09cb7245c5ea8010665e3f80c082"
Error: No tool or SDK found by name '2.0.20-lto'.
```

**Comment by kripken** [comment]
Is this using the very latest emsdk on the `main` branch?

**Comment by kripken** [comment]
If it's using an existing docker image, might need to pull the latest from github.

**Comment by donalffons** [comment]
For my local installation, I'm using [this arch package](https://archlinux.org/packages/community/x86_64/emscripten/).

For the Docker image (which I care more about), I just now made a Dockerfile with this content
```Dockerfile
FROM emscripten/emsdk:2.0.20 # also tried emscripten/emsdk:latest
```
and then I run
```
$ docker build --no-cache -t test .
```
> Sending build context to Docker daemon  2.048kB
> Step 1/1 : FROM emscripten/emsdk:2.0.20
>  ---> 50855adee81f
> Successfully built 50855adee81f
> Successfully tagged test:latest

```
$ docker run -it test 
```
> Setting environment variables:
> PATH = /emsdk:/emsdk/upstream/emscripten:/emsdk/node/14.15.5_64bit/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin

```
(inside container) $ emsdk install 2.0.20-lto
```
> Error: No tool or SDK found by name '2.0.20-lto'.

```
(inside container) $ emsdk update
```
> You seem to have bootstrapped Emscripten SDK by cloning from GitHub. In this case, use "git pull" instead of "emsdk update" to update emsdk. (Not doing that automatically in case you have local changes)

```
(inside container) $ emsdk update-tags
```
> `update-tags` is not longer needed.  To install the latest tot release just run `install tot`

**Comment by kripken** [comment]
Does it work if you do `git pull` as the tool suggests?

If not, what does `git show HEAD` show?

**Comment by donalffons** [comment]
...when I navigate into `cd /emsdk` in the docker container and run `git pull`, I get
> fatal: not a git repository (or any of the parent directories): .git

The folder /emsdk/.git exists inside the docker container, but it is empty (also contains no hidden files).

**Comment by kripken** [comment]
Yes, maybe try `git clone` and the standard steps from the emscripten download instructions.

I'm not sure how the docker container is set up, and why that doesn't just work. But maybe someone else here does.

**Comment by sbc100** [comment]
The docker container is like a pre-packaged version of emsdk.. its not designed such that you can install new SDK or upgrades.  We should make that more clear somehow.

**Comment by RReverser** [comment]
Can we perhaps publish LTO Docker images in addition to emsdk tags?
